src/App.tsx

```diff
@@ -28,16 +28,54 @@ const paddingValue = isMobile ? '42px' : '80px';
       <Typography variant="h1" gutterBottom>
       Oasis Playground
       </Typography>
-      <Typography gutterBottom sx={{color: 'white', fontSize: '20px', width: '100%', maxWidth: '750px', marginBottom: '24px'}} >
-      Discover the frontier of Web3 privacy through the projects and applications that are powered by the Oasis Network’s cutting-edge production-ready confidentiality technology. Become a contributor by adding your dApp to the Playground <Link href='https://github.com/oasisprotocol/playground/blob/main/CONTRIBUTING.md'  sx={{
-    color: 'white',
-    textDecorationColor: 'white',
-    '&:hover': {
-      color: 'white',
-    },
-  }} target="_blank" underline="always">here</Link>.
-      </Typography>
-      <Box>
+      <Grid container spacing={2} justifyContent="space-between" alignItems="flex-start" marginBottom={isMobile ? '36px' : '0'}>
+        <Grid item xs={12} md={9}>
+          <Typography gutterBottom sx={{ color: 'white', fontSize: '20px', width: '100%', maxWidth: '750px', marginBottom: isMobile ? '16px' : '24px' }}>
+            Discover the frontier of Web3 privacy through the projects and applications that are powered by the Oasis Network’s cutting-edge production-ready confidentiality technology. Become a contributor by adding your dApp to the Playground {' '}
+            <Link href='https://github.com/oasisprotocol/playground/blob/main/CONTRIBUTING.md' sx={{
+              color: 'white',
+              textDecorationColor: 'white',
+              '&:hover': {
+                color: 'white',
+              },
+            }} target="_blank" underline="always">
+              here
+            </Link>.
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Link
+            href='https://github.com/oasisprotocol/playground/blob/main/CONTRIBUTING.md'
+            target='_blank'
+            sx={{
+              display: 'block',
+              fontFamily: '"Figtree Variable", sans-serif',
+              borderRadius: '50px',
+              padding: '10px 25px',
+              fontWeight: '600',
+              marginTop: '8px',
+              textAlign: 'center',
+              border: '2px solid white',
+              width: isMobile ? '100%' : 'auto',
+              backgroundColor: 'transparent',
+              color: 'white',
+              fontSize: '16px',
+              textDecoration: 'none',
+              letterSpacing: '-0.03em',
+              cursor: 'pointer',
+              transition: 'all .2s ease',
+              '&:hover': {
+                backgroundColor: 'white',
+                color: '#3431AC',
+                border: '2px solid white'
+              },
+            }}
+          >
+            Add your dApp
+          </Link>
```

Comment:
I think you only need
```tsx
<Button
  href='https://github.com/oasisprotocol/playground/blob/main/CONTRIBUTING.md'
  target='_blank'
  color='inherit'
  variant='outlined'
  sx={{borderRadius: '30px'}}
>
  Add your dApp
</Button>
```

Reply:
I can keep it as a button, but still needs additional styling